docs/docs/smart-contracts/PSP1155/Extensions/psp1155burnable.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 3
+title: PSP1155 Burnable
+---
+
+This example shows how you can reuse the implementation of [PSP1155](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp1155) token with [PSP1155Burnable](https://github.com/Supercolony-net/openbrush-contracts/tree/main/contracts/token/psp1155/src/extensions/burnable.rs) extension.
+
+## How to use this extension
+
+For your smart contract to use this extension, you only need to implement the `PSP1155Burnable` trait in your `PSP1155` smart contract. Add import for `psp1155::extensions::burnable::*`, inherit the implementation for `PSP1155Burnable` trait, where you can also customize (override) the original functions from `PSP1155Burnable`.
```

Comment:
Could you add a link to `PSP1155` in our documentation, please? And do the same for another extensions)